splinepy/io/ioutils.py

```diff
@@ -97,3 +97,50 @@ def abs_fname(fname):
         fname = os.path.abspath(fname)
 
     return fname
+
+
+def strip_tabs(fname, overwrite=True, tab_expand=2):
+    """Replaces tabs in a text file with spaces
+
+    Parameters
+    ----------
+    fname : string
+      Filename
+    overwrite : bool
+      Inplace modification, otherwise new file with prefix copy is created
+    tab_expand : int
+      Determines how many spaces to be set for each tab
+
+    Returns
+    -------
+    None
+    """
+    if overwrite:
+        out_name = fname
+    else:
+        out_name = "copy." + fname
+    with open(fname) as inputFile:
+        file_contents = inputFile.read()
+        file_contents.replace("\t", " " * tab_expand)
```

Comment:
```suggestion
    file_contents.replace("\t", " " * tab_expand)
```

Reply:
Can also be here. Does not need to be in the file context. But can also stay inside.